Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Rifles/colony_rifles.yml

```
@@ -0,0 +1,52 @@
+- type: entity
+  parent: CMBaseWeaponRifle
+  name: AR10 Assault Rifle
+  id: WeaponRifleAR10
+  description: An earlier version of the more widespread M16 rifle. How one of these ended up here is a mystery of its own. Uses 7.62x51mm rounds.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Rifles/ar10.rsi
+    layers:
+    - state: base
+      map: [ "enum.GunVisualLayers.Base" ]
+    - state: mag-0
+      map: [ "enum.GunVisualLayers.Mag" ]
+    - sprite: _RMC14/Objects/Weapons/Guns/Attachments/stock.rsi
+      state: ar10_stock
+      offset: -0.5, 0
+  - type: Clothing
+    sprite: _RMC14/Objects/Weapons/Guns/Rifles/ar10.rsi
+  - type: ContainerContainer
+    containers:
+      gun_magazine: !type:ContainerSlot
+      gun_chamber: !type:ContainerSlot
+  - type: Gun
+    selectedMode: FullAuto
+    availableModes:
+    - SemiAuto
+    - Burst
+    - FullAuto
+    soundGunshot:
+      collection: CMAR10Shoot
+  - type: RMCSelectiveFire
+    baseFireModes:
+    - SemiAuto
+    - Burst
+    - FullAuto
+    recoilUnwielded: 3
+    scatterWielded: 3
+    scatterUnwielded: 11
+    baseFireRate: 2.5
+    burstScatterMult: 2
+  - type: ItemSlots
+    slots:
+      gun_magazine:
+        name: Magazine
+        insertSound: /Audio/_RMC14/Weapons/Guns/Reload/m41_reload.ogg
+        ejectSound: /Audio/_RMC14/Weapons/Guns/Reload/m41_unload.ogg
+        priority: 2
+        whitelist:
+          tags:
+          - RMCMagazineRifleAR10
+  - type: GunDamageModifier
+    multiplier: 1.1
```

From 13 this should be 1.4
```suggestion
    multiplier: 1.4
```